src/docker-osx-dev

```diff
@@ -409,7 +409,12 @@ function configure_docker_machine {
   DOCKER_MACHINE_DRIVER_NAME=$(inspect_docker_machine "{{.DriverName}}")
 
   DOCKER_HOST_SSH_URL="$DOCKER_HOST_USER@$DOCKER_HOST_IP"
-  DOCKER_HOST_SSH_KEY="$DOCKER_MACHINE_STORE_PATH/id_rsa"
+  DOCKER_HOST_SSH_KEY="$DOCKER_MACHINE_STORE_PATH/machines/$DOCKER_MACHINE_NAME/id_rsa"
+
+  if [[ ! -f $DOCKER_HOST_SSH_KEY ]]; then
```

Comment:
I can't find any documentation on that ever working on a previous version of docker-machine. It was in the original code.